docs/validators/key-management/multikey-nodes.md

```diff
@@ -0,0 +1,123 @@
+---
+id: multikey-nodes
+title: Multikey nodes management
+---
+
+[comment]: # (mx-abstract)
+This page contains information about how to manage multiple keys on an observing squad.
+
+[comment]: # (mx-context-auto)
+## Multikey architecture overview
+
+At the mainnet launch, and up until the release candidate RC/v1.6.0 becomes mainstream, each node could have managed only
+one key. So the relationship between the nodes and keys is `1:1`, the node "following" the shard where the managed key is assigned.
+The main idea behind the multikey feature is that a set of keys can be managed by an observing squad set. The same set of keys
```

Comment:
The main idea behind the multikey feature is that a set of keys can be managed by an observing squad set. The same set of keys should be provided for all observing squad nodes
=>
The multikey feature allows a node instance to hold more than one key. However, since MultiversX is a sharded blockchain and a single node is only able to store the state of a single shard at a time. Also, in each epoch, the nodes can be shuffled among shards. This means that running multiple keys will require at least 4 node instances (one for each shard + metachain), making it worthy only if one has more than 4 keys to manage. 
The same set of keys should be provided for all node instances

Reply:
rephrased